dist/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="utf-8" />
+        <title>Codidact - Helping each other learn.</title>
+        <meta
+            name="description"
+            content="Codidact is an open-source, community-controlled Q&amp;A platform. Currently under active development. Contribute by joining our team."
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+        <link rel="manifest" href="site.webmanifest" />
+        <link rel="apple-touch-icon" href="icon.png" />
+
+        <link rel="stylesheet" href="assets/css/normalize.css" />
+        <link rel="stylesheet" href="assets/css/primary.css" />
+    </head>
+
+    <body>
+        <!--[if IE]>
```

Comment:
This doesn't render on IE10 and IE11, right? ^^

Reply:
thats right - 10 and 11 wont render but everything down from there will